tests/e2e/tests/content-manager/preview.spec.ts

```diff
@@ -46,4 +47,26 @@ describeOnCondition(edition === 'EE')('Preview', () => {
 
     await expect(page.getByRole('link', { name: /open preview/i })).not.toBeVisible();
   });
+
+  test('Tabs for Draft and Publish should be visible for content type with D&P enabled', async ({
+    page,
+  }) => {
+    // Navigate to the Content Manager and open the edit view of a content type with D&P enabled
+    await clickAndWait(page, page.getByRole('link', { name: 'Content Manager' }));
+    await clickAndWait(page, page.getByRole('link', { name: 'Article' }));
+    await clickAndWait(page, page.getByRole('gridcell', { name: /west ham post match/i }));
+
+    // Check that preview opens in its own page
+    await clickAndWait(page, page.getByRole('link', { name: /open preview/i }));
+    // Draft status is visible (second Draft text is the draft tab)
+    await expect(page.getByText(/^Draft$/).nth(0)).toBeVisible();
+    await expect(page.getByRole('heading', { name: /west ham post match/i })).toBeVisible();
+
+    // Verify that Draft and Publish tabs are visible
+    await expect(page.getByText(/^Draft$/).nth(1)).toBeVisible();
+    await expect(page.getByText(/^Published$/)).toBeVisible();
```

Comment:
You can select by role here, which is better as it makes sure code is accessible too. There are tab, tablist, and tabpanel roles